gammapy/data/counts.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function, division
+from gammapy.spectrum import EnergyBinning
+
+
+__all__ = ['CountsSpectrumDataset',
+           ]
+
+
+class CountsSpectrumDataset(object):
+    """
+    Counts spectrum dataset (PHA wrapper).
+    
+    Test case for EnergyBinning class
+    """
+  
+    def __init__(self, counts, energy):
+
+        #does not work (why?) -> Kind of crucial
```

Comment:
Remove `isinstance` check here?
If we do it here, we should probably do it in 1000 other places or better yet, go to a statically typed language like C++, no?